src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -329,13 +329,66 @@ public void getProjectSitePages(@PathVariable String site_id,
 						+ e.getMessage();
 				log.error(errorMessage);
 			}
+		
+			pagesString = siteHasContentItems(site_id, pagesString, sessionId);
 		}
 		
 		rv.put("pagesString", pagesString);
 		rv.put("errorMessage", errorMessage);
 		rv.put("requestUrl", requestUrl);
 		return rv;	
 	}
+	
+	/**
+	 * adds an extra "hasContentItem" JSON element to the site tool JSON feed. 
+	 * True if site has content resources; false if the site content is empty.
+	 * @param site_id
+	 * @param pagesString
+	 * @param sessionId
+	 * @return
+	 */
+	private String siteHasContentItems(String site_id, String pagesString,String sessionId) {
+		JSONArray rvSitePagesArray = new JSONArray() ;
+		
+		RestTemplate restTemplate;
+		JSONArray pages = new JSONArray(pagesString);
+		// iterate through all pages
+		for (int iPage = 0; iPage < pages.length(); ++iPage) {
+		    JSONObject page = pages.getJSONObject(iPage);
+		    if (page.has("tools"))
+		    {
+				// iterate through all tools within page
+		    	JSONArray tools = (JSONArray) page.get("tools");
+		    	for (int iTool = 0; iTool < tools.length(); ++iTool) {
+				    JSONObject tool = tools.getJSONObject(iTool);
+				    if (tool.has("toolId") && "sakai.resources".equals(tool.get("toolId")))
+				    {
+				    	// found Resource tool
+				    	restTemplate = new RestTemplate();
+						String resourceToolRequestUrl = env.getProperty("ctools.server.url")
```

Comment:
Yes, I would expect them to be, too.  But what if somebody forgot to include it in the configuration file or misspelled (or mis-capitalized) the name?

If this URL string turns out to be wrong because of misconfiguration, will the problem be handled by `restTemplate.getForObject` when it's called with that wrong URL?